vm/policy.go

```diff
@@ -121,7 +122,8 @@ func PolicyHandling(t string, path string, o PolicyOptions, httpAddress string,
 	policies := strings.Split(string(policyFile), "---")
 
 	for _, policy := range policies {
-		if policy == "" {
+
+		if matched, _ := regexp.MatchString("^\\s*$", policy); matched {
```

Comment:
What do you think about using https://pkg.go.dev/unicode#IsSpace instead?

Reply:
After reading the `unicode.IsSpace()` function .. I see that I takes rune as a single character . Here in the for loop we are dealing with strings. Even if we try to convert the string to rune slice we cannot pass it to the `unicode.IsSpace()`  function .
Imo regex seems fine.